File: Ass4/ass4src/java/xmlParser/src/Driver.java
import java.util.Iterator;
import java.util.Vector;

import dataStructure.Employee;


public class Driver {

	public static void main(String args[]){
		XMLParser xmlParser = new XMLParser("src/employees.xml");
		xmlParser.parseXmlFile();
		xmlParser.parseDocument();
		Vector<Employee> employees = xmlParser.getParsedXML();
		
		printData(employees);
		
	}
	
	
	private static void printData(Vector<Employee> employees){

		System.out.println("Number of Employees '" + employees.size() + "'.");

		Iterator<Employee> it = employees.iterator();
		while(it.hasNext()) {
			System.out.println(it.next().toString());
		}
	}
}

File: Ass4/ass4src/c++/XML employee example/Driver.cpp
#include "Driver.h"

int main(int argc, char** argv){

	PocoXMLParser xmlParser("employees.xml");
	xmlParser.parseXmlFile();
	xmlParser.parseDocument();
	std::vector<Employee*> employees = xmlParser.getParsedXML();
	printData(employees);
	return 0;
}


void printData(std::vector<Employee*> employees){

	std::cout << "Number of Employees '" << employees.size() << "'." << std::endl;

	std::vector<Employee*>::iterator it;
	for (it=employees.begin(); it!=employees.end(); it++)
		std::cout << (*it)->toString();
}
